### PR TITLE
Adding driver license recipe [#150016263]

### PR DESCRIPTION
**Pull request by aboone**

### CODE REVIEW

File: recipes/recipe-16-driver-license.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+ * `attribute.property.addressCity`
+ * `attribute.property.addressLine1`
+ * `attribute.property.addressLine2`
+ * `attribute.property.addressStateCode` (a 2-character state code - example: `FL`)
+ * `attribute.property.addressZIP4`
+ * `attribute.property.addressZIP5`
+ * `attribute.property.documentCategory` (possible values: `DRIVER_LICENSE`, `PERMIT_ID_CARD`, `STATE_ID_CARD`)
+ * `attribute.property.driverLicenseExpirationDate` (`YYYY-MM-DD` format)
+ * `attribute.property.personBirthDate` (`YYYY-MM-DD` format)
+ * `attribute.property.personEyeColor` (possible values: `BLK` = Black, `BLU` = Blue, `BRO` = Brown, `DIC` = Dichromatic, `GRY` = Gray, `GRN` = Green, `HAZ` = Hazel, `MAR` = Maroon, `PNK` = Pink)
```
Comment (coatsbj) [inline recipes/recipe-16-driver-license.md]: There is one additional option, 'UNK'='Unknown'...not sure how that one is handled though.

File: recipes/recipe-16-driver-license.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+ * `attribute.property.personFirstName`
+ * `attribute.property.personHeight` (represented as an integer, the number of feet and inches - example: `510` to represent 5' 10")
```
Comment (coatsbj) [inline recipes/recipe-16-driver-license.md]: Has this always been the case?  I thought it was indeed a string (5' 10")...

File: recipes/recipe-16-driver-license.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+ * `attribute.property.personLastName`
+ * `attribute.property.personMiddleName`
+ * `attribute.property.personNameSuffix`
+ * `attribute.property.personSexCode` (`MALE` or `FEMALE`)
```
Comment (coatsbj) [inline recipes/recipe-16-driver-license.md]: I was pretty sure this was 'M' or 'F'...am I wrong on that one as well?

File: recipes/recipe-16-driver-license.md (hunk 1 above)
Comment (aboone) [reply recipes/recipe-16-driver-license.md]: `UNK` appears to be unsupported, I get the following error:

```
{
    "httpStatus": 400,
    "message": "No enum constant org.aamva.niem.extensions._1.PersonEyeColorCodeSimpleType.UNK",
    "status": "400"
}
```

File: recipes/recipe-16-driver-license.md (hunk 2 above)
Comment (aboone) [reply recipes/recipe-16-driver-license.md]: Yes, we need an integer. I got a 500 error when supplying the height as a formatted string like you have shown

File: recipes/recipe-16-driver-license.md (hunk 3 above)
Comment (aboone) [reply recipes/recipe-16-driver-license.md]: We need the full string, this is the result with just M:

```
{
    "httpStatus": 400,
    "message": "No enum constant com.sheerid.thirdparty.aamva.model.SexCode.M",
    "status": "400"
}
```

**Pull request merged**